libs/api/books/src/lib/reading-list.controller.ts

```diff
@@ -20,4 +20,14 @@ export class ReadingListController {
   async removeFromReadingList(@Param() params) {
     return await this.readingList.removeBook(params.id);
   }
+
+  @Post('/reading-list/:id/finished')
```

Comment:
This isn't what the spec requested is it?

Reply:
no, should be `PUT`.